Sarah Hong/Homework/Week-01/Wednesday/Arrays & Functions /js/homework_02.js

```diff
@@ -0,0 +1,114 @@
+// # Array and Functions Bonus Material
+//
+// 1. Define a function `maxOfTwoNumbers` that takes two numbers as arguments and returns the largest of them. Use the if-then-else construct available in Javascript. You'll have to remember your pre-work, or do some googling to figure this out.
+
+const maxOfTwoNumbers = function ( x, y ) {
+      if ( x > y ) {
+      return x;
+    } else {
+      return y;
+    }
+  }
+
+console.log( maxOfTwoNumbers(23, 87) );
+console.log( maxOfTwoNumbers(89, 65) );
+console.log( maxOfTwoNumbers(8, 5) );
+
+// 2. Define a function `maxOfThree` that takes three numbers as arguments and returns the largest of them.
+//
+// RESEARCH : Spread operator, sort method, array.pop
+// RESEARCH : maxOfThree.sort();
+// MISTAKES REPEATED : differenciate || and &&.
+
+const maxOfThree = function ( x, y, z ) {
+    if ( x > y && x > z ) {
+      return x;
+  } else if ( y > x && y > z ) {
+      return y;
+  } else {
+      return z
+    }
+  }
+
+console.log( maxOfThree( 5, 8, 18 ) );
+console.log( maxOfThree( 98, 2, 13 ) );
+console.log( maxOfThree( 63, 48, 33 ) );
+console.log( maxOfThree( 124, 346, 235 ) );
+console.log( maxOfThree( 1, 4, 12 ) );
+
+// 3. Write a function that takes a character (i.e. a string of length 1) and returns true if it is a vowel, false otherwise.
+//
+// RESEARCH : logical or operators. Create array of vowels. Run .includes in array.
+
+const vowels = function ( character ) {
+  if ( character === 'a' ) {
+    return true;
+  } else if ( character === 'e' ) {
+    return true;
+  } else if ( character === 'i') {
+    return true;
+  } else if ( character === 'o' ) {
+    return true;
+  } else if ( character === 'u' ) {
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
This looks good. If you wanted to make this shorter you could do something like: 
const vowels = function ( character ) {
  if ( character === 'a' ||  character === 'e' || character === 'i' || character === 'o' || character === 'u' ) {
    return true;
  } else {
    return false;
  }
}

OR as the instructions suggest create an array and loop through that. (I'll leave that one to you if you want to give that one another go)

Reply:
Thanks Camilla! 
I've tried the shortened version you mentioned above before, but everything console.logged to all 'true' or all 'false'. I'll try again - I must have added something different that made it not work. 
And yes, ill have a crack on the array and loop solution and let you know how I go :)